Specify the output file for lcov --remove as an absolute path.

aminiussi opened the pull request:
For some reason,` lcov --remove ... --outputfile <file> `will try to open `<file>` wrt lcov location (I might have something to do with having . in your PATH).

There are two possible workarounds for this problem
  * do not use `--output-file` but redirect the output instead
  * explicitly put that file into **${CMAKE_BINARY_DIR}** (or another directory, maybe **${CMAKE_CURRENT_BINARY_DIR}** ?)

bilke commented:
The commands are already executed with a working directory of `${PROJECT_BINARY_DIR}` which should be equal to `${CMAKE_BINARY_DIR}`:

```cmake
WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
```

aminiussi commented:
Note that, for some reason, the lcov .. --output-file problem is not systematic, 
 * Without the modification:
```
/usr/bin/lcov --remove fargocover.info '/opt/*' '/usr/*' --output-file fargocover.info.cleaned
Reading tracefile fargocover.info
lcov: WARNING: negative counts found in tracefile fargocover.info
[...]
Writing data to fargocover.info.cleaned
lcov: ERROR: cannot write to fargocover.info.cleaned!
CMakeFiles/fargocover.dir/build.make:57: recipe for target 'CMakeFiles/fargocover' failed
```
 * With the modification:
```
/usr/bin/lcov --remove fargocover.info '/opt/*' '/usr/*' --output-file /home/alainm/views/fargOCA/dbg/fargocover.info.cleaned
Reading tracefile fargocover.info
lcov: WARNING: negative counts found in tracefile fargocover.info
[...]
Writing data to /home/alainm/views/fargOCA/dbg/fargocover.info.cleaned
Summary coverage rate:
  lines......: 50.8% (3448 of 6787 lines)
  functions..: 77.2% (366 of 474 functions)
  branches...: no data found
/usr/bin/genhtml -o fargocover /home/alainm/views/fargOCA/dbg/fargocover.info.cleaned
[...]
```

bilke commented:
Ah ok, maybe it works now because of the path to the output file is now an absolute path.
